Automattic/jetpack
# [Boost] Remove untranslated strings.

Fixes https://github.com/Automattic/boost-cloud/issues/318

Fixes untranslatable strings, by removing one file that is no longer used and by making another translatable.

## Proposed changes:

* Remove an unused page
* Make a string translatable.

### Other information:


## Jetpack product discussion


n/a

## Does this pull request change what data or activity we track or use?


no

## Testing instructions:





* View an image report
* Ensure its title looks correct.
